database/migrations/2017_07_12_143016_create_foreign_keys.php

```diff
@@ -13,6 +13,8 @@ class CreateForeignKeys extends Migration
      */
     public function up()
     {
+        Schema::disableForeignKeyConstraints();
```

Comment:
Do we not do this globally?

Reply:
Nope, foreign keys are enabled during migrations